plugin/mayaHydraSceneBrowser/CMakeLists.txt

```diff
@@ -64,6 +64,17 @@ maya_set_plugin_properties(${TARGET_NAME})
 if(IS_MACOSX OR IS_LINUX) 
     mayaUsd_init_rpath(rpath "lib/maya")
     mayaUsd_add_rpath(rpath "${CMAKE_INSTALL_PREFIX}/lib")
+    if(DEFINED MAYAHYDRA_TO_USD_RELATIVE_PATH)
+        mayaUsd_add_rpath(rpath "../../${MAYAHYDRA_TO_USD_RELATIVE_PATH}/lib")
+    elseif(DEFINED PXR_USD_LOCATION)
+        mayaUsd_add_rpath(rpath "${PXR_USD_LOCATION}/lib")
+    endif()
+    if (IS_LINUX AND DEFINED MAYAHYDRA_TO_USD_RELATIVE_PATH)
+        mayaUsd_add_rpath(rpath "../../${MAYAHYDRA_TO_USD_RELATIVE_PATH}/lib64")
+    endif()
+    if(IS_MACOSX AND DEFINED MAYAHYDRA_TO_USD_RELATIVE_PATH)
+        mayaUsd_add_rpath(rpath "../../../../Maya.app/Contents/MacOS")
+    endif()
```

Comment:
This is copy-pasted from https://github.com/Autodesk/maya-hydra/blob/dev/lib/mayaHydra/mayaPlugin/CMakeLists.txt#L88-L98